pyface/ui/qt4/code_editor/pygments_highlighter.py

```diff
@@ -21,6 +21,11 @@ def get_tokens_unprocessed(self, text, stack=('root',)):
     """ Split ``text`` into (tokentype, text) pairs.
 
         Monkeypatched to store the final stack on the object itself.
+
+        The `text` parameter this gets passed is only the current line, so to
```

Comment:
Changed this, that is out of scope of this project :)
Though i believe both usages are correct.